ui/app/components/pending-tx.js

```diff
@@ -57,7 +57,7 @@ PendingTx.prototype.render = function () {
   const safeGasLimit = safeGasLimitBN.toString(10)
 
   // Gas Price
-  const gasPrice = txParams.gasPrice || MIN_GAS_PRICE_BN.toString(16)
+  const gasPrice = txParams.gasPrice || MIN_GAS_PRICE_BN.toString(10)
```

Comment:
why is the decimal `toString(10)` going into the `hexToBn`?

Reply:
Will fix